src/player.js

```diff
@@ -1701,6 +1720,10 @@ export default class Player extends FakeEventTarget {
         if (browser === 'Edge' || browser === 'IE') {
           this._removeTextCuePatch();
         }
+        if (this._engine && this._pendingSelectedVideoTrack) {
+          this._engine.selectVideoTrack(this._pendingSelectedVideoTrack);
```

Comment:
clearer to do this on `PLAY` or `PLAYING` no?

Reply:
done